contributing.md

```diff
@@ -109,3 +109,15 @@ For example, the following test method names could be used:
 
 - `GetCollections_ShouldSucceed()`
 - `GetCollections_ShouldThrow_WhenDatabaseIsOffline()`
+
+## Releases
+
+To create a new release, follow these steps:
+
+1. Pull branch for release from actify-inc repo.
+2. Check the version number in ./arangodb-net-standard/ArangoDBNetStandard.csproj file
+  - if the version number is incorrect, create a separate commit to change the version number, push it and get it merged before proceeding.
+3. Run `dotnet pack --configuration Release ./arangodb-net-standard/ArangoDBNetStandard.csproj`
+4. Upload the resultant nuget package via https://www.nuget.org/packages/manage/upload
```

Comment:
I think we need to use a specific account to do that step. Should we explain that?

Reply:
I mostly put this up for my own benefit but yeah it's true you might need to log in as an "owner" to update the package.